test/it/006activityLogTest.js

```diff
@@ -1,28 +1,32 @@
 import * as R from 'ramda'
 import { expect } from 'chai'
-import { uuidv4 } from '@core/uuid'
 
-import * as Survey from '@core/survey/survey'
-import * as Record from '@core/record/record'
-import * as ObjectUtils from '@core/objectUtils'
+import { uuidv4 } from '../../core/uuid'
```

Comment:
do we need relative imports in test? they can be compiled.. or not? let's talk :p

Reply:
uhm, ok!